styles/bsky/catppuccin.user.css

```diff
@@ -565,6 +578,26 @@
     [style*="background-color: rgb(185, 185, 193)"] {
       background-color: @overlay2 !important;
     }
+    
+    /* skeletons */
+    .r-kdyh1x, .r-cpet4d {
```

Comment:
are these classes the only way to reliably match skeletons? i've seen these classes change even from small changes, so i'd prefer if there was a more reliable way to match these
(if not then thats okay, we can just update the classes when they (inevitably) break like i do with most of the inline colors already anyways)

Reply:
havent found any really :c

their color is the same as "secondary background color" which is already overridden above but looks pretty bad:
<img width="611" alt="image" src="https://github.com/user-attachments/assets/f4dec89c-b9a2-4a28-af04-877064d926a5">

these classes set the "border-radius" values on the skeletons which *seem* to only be used on them, so combined with the color this should be safe enough

Reply:
alright ty !!